server/swap/swap.go

```diff
@@ -434,6 +425,74 @@ func (s *Swapper) processBlock(block *blockNotification) {
 	}
 }
 
+func (s *Swapper) redeemStatus(match *matchTracker) (makerRedeemComplete, takerRedeemComplete bool) {
+	mStatus, tStatus := match.makerStatus, match.takerStatus
+	if !mStatus.redeemTime.IsZero() {
+		confs, err := mStatus.redemption.Confirmations()
+		if err != nil {
+			log.Errorf("Confirmations failed for maker redemption %v: err",
+				mStatus.redemption.TxID(), err) // Severity?
```

Comment:
I think we'll need to keep and eye out for this because it's not clear to me how to handle such an error.